src/pages/ProductDetail/BeforeCheck/BeforeCheck.js

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { FiChevronDown } from 'react-icons/fi';
+import { FiChevronUp } from 'react-icons/fi';
+
+function BeforeCheck() {
+  const [toggle, setToggle] = useState([false, false, false]);
+
+  function handleToggle(targetId) {
+    const newToggle = [...toggle];
+    newToggle[targetId - 1] = !newToggle[targetId - 1];
+    setToggle([...newToggle]);
+  }
+
+  return (
+    <div>
+      <Title>구매 전 꼭 확인해주세요!</Title>
+      {CONFIRM.map((data, i) => {
+        return (
+          <Guide key={data.id}>
```

Comment:
```suggestion
          <div key={data.id}>
```
따로 스타일 적용 안 하시는 거면 스타일 컴포넌트로 하지 않으셔도 됩니다!